gce-jetty/src/main/docker/Dockerfile

```diff
@@ -27,11 +27,14 @@ RUN mv /usr/local/jetty-distribution-* $JETTY_HOME && chown -R jetty:jetty $JETT
 ENV JETTY_BASE /var/lib/jetty
 RUN mkdir -p $JETTY_BASE
 WORKDIR $JETTY_BASE
-RUN java -jar $JETTY_HOME/start.jar --add-to-startd=http,deploy,setuid && \
+RUN java -jar $JETTY_HOME/start.jar --add-to-startd=http,deploy,jsp,jstl,setuid && \
     chown -R jetty:jetty $JETTY_BASE && \
-    mkdir /tmp/jetty
+    mkdir /tmp/jetty && chown -R jetty:jetty /tmp/jetty
+
+RUN ln -s $JETTY_BASE/webapps/root /app
+WORKDIR /app
```

Comment:
For the V1 compat mode, I think that is what we have specified. So yes, for
compat.
I guess, I can set the WORKDIR only in the jetty9-compat image.

On Thu, Sep 3, 2015 at 6:03 PM, Greg Wilkins <notifications@github.com>
wrote:

> In gce-jetty/src/main/docker/Dockerfile
> <https://github.com/GoogleCloudPlatform/appengine-java-vm-runtime/pull/59#discussion_r38713593>
> :
>
> >      chown -R jetty:jetty $JETTY_BASE && \
> > -    mkdir /tmp/jetty
> > +    mkdir /tmp/jetty && chown -R jetty:jetty /tmp/jetty
> > +
> > +RUN ln -s $JETTY_BASE/webapps/root /app
> > +WORKDIR /app
>
> Why make the workdir /app?
> This is a deviation from the official jetty-docker image that may hurt
> portability of webapps (albeit badly written ones that assume working dir).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/appengine-java-vm-runtime/pull/59/files#r38713593>
> .
>